packages/cli/CONTRIBUTING.md

```diff
@@ -1,22 +1,22 @@
-# Contributing to Teams Toolkit CLI 
+# Contributing to Microsoft 365 Agents Toolkit CLI 
 
-Welcome, and thank you for your interest in contributing to Teams Toolkit CLI!
+Welcome, and thank you for your interest in contributing to Microsoft 365 Agents Toolkit CLI!
 
-Please review this document for setting up your development environment, debugging and run Teams Toolkit CLI. If you have any questions, please raise your issue on github.
+Please review this document for setting up your development environment, debugging, and run Microsoft 365 Agents Toolkit CLI. If you have any questions, please raise your issue on github.
 
 ## Prerequisites
 ---
 
-Verify you have the right prerequisites for building Teams apps:
+Verify you have the right prerequisites for building Microsoft 365 Apps:
```

Comment:
M365 apps, teams apps and agents? Add @summzhan to confirm here.

Reply:
Can we first merge this PR for now and discuss the issue next time.